Mathlib/Data/IsROrC/Basic.lean

```diff
@@ -181,7 +181,7 @@ theorem ofReal_inj {z w : ℝ} : (z : K) = (w : K) ↔ z = w :=
 #align is_R_or_C.of_real_inj IsROrC.ofReal_inj
 
 set_option linter.deprecated false in
-@[deprecated, isROrC_simps] -- porting note: was `simp` but `simp` can prove it
+@[deprecated, isROrC_simps] -- porting note (#10745): was `simp` but `simp` can prove it
```

Comment:
This looks more like a previous `@[simp]` annotation:
```suggestion
@[deprecated, isROrC_simps] -- porting note (#10618): was `simp` but `simp` can prove it
```
Same for the rest in this file.